spring-boot-project/spring-boot-autoconfigure/src/test/resources/org/springframework/boot/autoconfigure/hazelcast/hazelcast-specific.xml

```diff
@@ -2,16 +2,13 @@
 		   xmlns="http://www.hazelcast.com/schema/config"
 		   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
 
-	<queue name="foobar"/>
```

Comment:
Same remark. Is this related to the main purpose of this PR?

Reply:
Reverted.